edx-platform/msus-produhktiv/lms/templates/header/header.html

```diff
@@ -92,4 +114,4 @@
             </label>
         </form>
     % endif
-% endif
+% endif
```

Comment:
@jignaciopm there's missing a new line at the end of the file

Reply:
@amalbas ready on my last commit